src/app/chat/chat-room/chat-room.component.ts

```diff
@@ -19,57 +19,60 @@ interface Chat {
   templateUrl: "./chat-room.component.html",
   styleUrls: ["./chat-room.component.scss"]
 })
-export class ChatRoomComponent implements OnInit {
+export class ChatRoomComponent implements OnInit, AfterContentInit {
   // @TODO need to create method to convert chat time to local time.
   @ViewChild(Content) content: Content;
 
   message: any;
   messageList: any[];
   selectedChat: Chat;
   chatColors: any[];
-  routeTeamId:number = 0;
-  routeTeamMemberId:number = 0;
-  messagePageNumber:number = 0;
-  messagePagesize:number = 20;
+  routeTeamId: number = 0;
+  routeTeamMemberId: number = 0;
+  messagePageNumber: number = 0;
+  messagePagesize: number = 20;
 
   constructor(
     private chatService: ChatService,
     private router: Router,
     private storage: BrowserStorageService,
     private activatedRoute: ActivatedRoute
-  ) {
+  ) {}
+
+  ngAfterContentInit() {
+    this.content.scrollToBottom();
   }
 
   ngOnInit() {
     this.chatColors = this.storage.get("chatAvatarColors");
     this.selectedChat = this.storage.get("selectedChatObject") || {
```

Comment:
Why do we have to get & set the chat object in local storage? 
Can we pass those as parameters in the route of this page?
Note:
Think about how we implement direct link to the chat room page.

Reply:
i try some options,
1.  passing data using `this.router.navigate`. method using now.
2. passing data using `this.navCtrl.navigateForward`.
in both option to pass chat object i have use `JSON.stringify()` method to before add chat object to param. and add `JSON.parse()` method to access data. but issue is after `JSON.stringify()` and set to parameters chat object data showing on the URL. 
BUT I have use that method because URL not showing the PWAs. but if some one use browser it will show the chat object data.